topup.go

```diff
@@ -50,8 +49,13 @@ type Topup struct {
 	FailureMessage           string              `json:"failure_message"`
 	ID                       string              `json:"id"`
 	Livemode                 bool                `json:"livemode"`
+	Metadata                 map[string]string   `json:"metadata"`
+	Object                   string              `json:"object"`
```

Comment:
I think this one is an enum with just one value `topup` for now, right? Do we want to add that one value as TopupObject or something?

Reply:
We've never done this so no. It's really just a string and I've added it manually to resources when I catch it.